Mage.Sets/src/mage/cards/k/KujaGenomeSorcerer.java

```diff
@@ -0,0 +1,68 @@
+package mage.cards.k;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.condition.Condition;
+import mage.abilities.condition.common.PermanentsOnTheBattlefieldCondition;
+import mage.abilities.decorator.ConditionalOneShotEffect;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.TransformSourceEffect;
+import mage.abilities.hint.ConditionHint;
+import mage.abilities.hint.Hint;
+import mage.abilities.keyword.TransformAbility;
+import mage.abilities.triggers.BeginningOfEndStepTriggeredAbility;
+import mage.constants.ComparisonType;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.filter.common.FilterControlledPermanent;
+import mage.game.permanent.token.BlackWizardToken;
+
+/**
+ * @author balazskristof
+ */
+public final class KujaGenomeSorcerer extends CardImpl {
+
+    private static final Condition condition = new PermanentsOnTheBattlefieldCondition(
+            new FilterControlledPermanent(SubType.WIZARD, "Wizards"), ComparisonType.OR_GREATER, 4
+    );
+    private static final Hint hint = new ConditionHint(condition, "You control 4 or more Wizards");
```

Comment:
sorry, I should specified to use a `ValueHint` with `PermanentsOnBattlefieldCount`